internal/cluster/validator.go

```diff
@@ -465,3 +466,28 @@ func (v *clusterValidator) printNtpServerConfigured(c *clusterPreprocessContext,
 		return fmt.Sprintf("Unexpected status %s", status)
 	}
 }
+
+func (v *clusterValidator) allOperatorsMinRequirementsSatisfied(c *clusterPreprocessContext) validationStatus {
+	cluster := &models.Cluster{}
+	// check for all the operators here.
+	if cluster.Operators == nil || cluster.Operators.Ocs == nil {
+		return boolValue(true)
+	}
+	if !*cluster.Operators.Ocs.Enabled {
```

Comment:
it's not safe to use points this way
```suggestion
	if !swag.BoolValue(cluster.Operators.Ocs.Enabled) {
```